src/services/degreeService.ts

```diff
@@ -0,0 +1,37 @@
+import { Degree } from '../typeDefs/gqlTypes';
+import {Degree as PrismaDegree } from '@prisma/client';
+
+import prisma from '../db/client';
+
+function convertPrismaToGraphQLDegree(input: PrismaDegree | null) {
+    if (!input) { return null; }
+    return {
+        id: String(input.id),
+        nameJa: input.nameJa,
+        nameEn: input.nameEn,
+        abbreviation: input.abbreviation
+    } as Degree;
+}
+
+// TODO: add a validation step for incoming parameters
+export const getDegreeById = async (id: string) => {
+    const degree = await prisma.degree.findUnique(
```

Comment:
future ticket: we should add in some form of data validation across all of the services so people can't sql inject.

Reply:
Yes... we should also check if prisma has any built-in injection prevention-- some ORMs do

Please add to #46